### DISCUSSION TITLE
Are experimental Custom Routes currently supported on ZEIT Now?

### DISCUSSION THREAD

**Discussion by borekb** [vercel/next.js#10695] [Help] [answered]
I know it's still an experimental feature but the build output shows "sources" and "destination" in our case, though we don't see the correct routing behavior.

(Sorry if this is not the best place to post this question; I'm just keen to try out Discussions 😄.)

**Comment by Timer** [comment]
Can you please update the title to be "Are Experimental Custom Routes currently supported on ZEIT Now?"

**Reply by borekb** [reply]
Done.

**Comment by Timer** [comment, marked as answer]
The experimental custom routes feature only works with non-dynamic pages on ZEIT Now currently.

They will be fully compatible when the feature is no longer experimental.
